lib/upfluence/error_logger/sentry.rb

```
@@ -64,6 +82,7 @@ def notify(error, *args)
           end
 
           scope.set_extras(prepare_extra(context))
+          set_error_level(scope, error)
```

could we do that in a event_processor? (say in the block L50?) ignored exception could be filtered first